app/controllers/xapi/DocumentController.php

```diff
@@ -83,6 +83,8 @@ public function store() {
     $data['content_info'] = $this->getAttachedContent('content');
     $data['ifMatch'] = \LockerRequest::header('If-Match');
     $data['ifNoneMatch'] = \LockerRequest::header('If-None-Match');
+    $data['ifMatch'] = isset($data['ifMatch']) ? trim($data['ifMatch'], '"') : null;
+    $data['ifNoneMatch'] = isset($data['ifNoneMatch']) ? trim($data['ifNoneMatch'], '"') : null;
```

Comment:
I'm aware of this already. Once the ETags are wrapped in quotes for the xAPI test suite I'll change this. For now though, quotes can't appear in the hash, so despite trimming additional quotes the world will probably spin on.